Core/CoreTests/Planner/CalendarViewControllerTests.swift

```diff
@@ -84,6 +84,12 @@ class CalendarViewControllerTests: CoreTestCase, CalendarViewControllerDelegate
         XCTAssertEqual(controller.daysHeight.constant, 48)
 
         controller.showDate(DateComponents(calendar: .current, year: 2020, month: 2, day: 1).date!)
+
+        //  wait for set month animation
+        let animationExpectation = XCTestExpectation(description: "")
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) { animationExpectation.fulfill() }
+        wait(for: [animationExpectation], timeout: 0.5)
```

Comment:
Since the timeout and the success time are both 0.5, this might be flaky. Also, if what needs to happen is letting the main queue run, that can be done more directly with `RunLoop.current.run(until: Date() + 0.5)`